### PR TITLE
RFC and README draft

### PR DESCRIPTION
**Pull request by k8hertweck**
Anything to change or add, @mkuzak and @ChristinaLK ?

@tracykteal could you take a quick look (particularly the staff RFC) and see if there is anything that is grossly incorrect?

Other things to do: blog post (trim down the README) and flesh out issues (include links to the relevant RFCs).

### CODE REVIEW

File: RFCs/RFC4.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+* Additional responsibilities to consider for additional staff hires or staff reorganization.  Some of these are 
+“unofficial” duties for current SWC or DC staff; others are done by volunteers from the community.
+  * Personnel operations (hiring, performance evaluations, professional development)
+  * Technical operations (managing website, sysadmin for servers)
+  * Lesson operations (lesson templating, lesson release)
+
+## For Discussion:
+
+Please add your comments on RFC1 to [this issue](#4).
+* Q1: Do you have any comments or suggestions on the role of staff in the context of Carpentries umbrella organisation?
```
Comment (ChristinaLK) [inline RFCs/RFC4.md]: The Carpentries

**Comment by ChristinaLK** [comment]
This all looks good to me!

**Comment by k8hertweck** [comment]
Thanks @ChristinaLK !

File: RFCs/RFC3.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+[SWC Advisory Council](https://github.com/swcarpentry/board/blob/master/advisory-council.md), with each member 
+organization assigning a point of contact to serve on the Council. 
+* Responsibilities: The Membership Council will serve as a method with which to communicate with institutional 
+members about changes in policy related to financial stakeholders, as well as a mechanism for organizational members 
+to raise issues with both staff and The Carpentries Board of Directors. The Membership Council will ideally be developed 
+into a stronger group of advocates for The Carpentries through regular meetings and information sharing.
+* Medium to long-term plan: Create a separate Advisory Council (8-10 individuals) of non-Board members who can provide 
+outside insight into strategic planning.
+
+## For discussion:**
```
Comment (mkuzak) [inline RFCs/RFC3.md]: asterisks's here at the end of the line are unnecessary

File: RFCs/RFC4.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  goals of lesson organizations
+  * Ensuring human resources requirements for all staff are met, including regular performance evaluations, professional 
+  development opportunities, and hiring additional staff.
+* Director of Business Development: Business development is critical to the long-term sustainability of the organization. 
+This Director will manage all new contract negotiations and renewals with existing member organizations, and will report 
+to the ED.
+* Other staff: All other current staff will be invited to transition to employees of The Carpentries 
+(note: Maneesha, Erin, and Belinda already share responsibility with for both SWC and DC). Specific responsibilities will 
+be determined by the ED, and will include having at least one staff person assigned as point of contact for each of the 
+subcommittees (see the [RFC on subcommittees]()).
```
Comment (mkuzak) [inline RFCs/RFC4.md]: should point here RFCs/RFC6.md

File: RFCs/RFC3.md (hunk 2 above)
Comment (mkuzak) [inline RFCs/RFC3.md]: remove trailing asterisks's

File: RFCs/RFC4.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  * Community engagement/management
+* Additional responsibilities to consider for additional staff hires or staff reorganization.  Some of these are 
+“unofficial” duties for current SWC or DC staff; others are done by volunteers from the community.
+  * Personnel operations (hiring, performance evaluations, professional development)
+  * Technical operations (managing website, sysadmin for servers)
+  * Lesson operations (lesson templating, lesson release)
+
+## For Discussion:
+
+Please add your comments on RFC1 to [this issue](#4).
```
Comment (mkuzak) [inline RFCs/RFC4.md]: comments on RFC4

File: RFCs/RFC7.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  including regular reporting on activities.
+  * Curriculum development: What is a workshop for the Lesson Organization?
+  * Lesson maintenance: What is lesson life cycle (goals of lesson, when does it become supported, when is it retired), 
+  maintaining lesson repositories in GitHub based on the lesson example from The Carpentries GitHub organization, 
+  determining processes associated with lesson maintenance (recommendations for lesson incubation and maintenance?)
+* Governance: Each lesson organization should have a point of contact (preferably a committee of 5-7 people led by a Chair), 
+although the specific methods through which leadership occurs is determined by the lesson organization.
+Onboarding for new lesson organizations: The Carpentries Board will be responsible for assessing alignment of the prospective 
+lesson organization’s mission and their ability to meet the responsibilities described. Given these decisions impact 
+organizational resources, the Board of Directors will solicity community input prior to deciding whether to extend an offer.
```
Comment (mkuzak) [inline RFCs/RFC7.md]: solicity > solicit

**Review by mkuzak — COMMENTED** [review]
All good. I picked some places which require small fixes.

**Comment by k8hertweck** [comment]
Thanks @mkuzak! I'll make this changes; would you be able to add some additional text to the issues (I created them as placeholders so I could add links to the RFCs)

**Comment by k8hertweck** [comment]
Comments from @mkuzak have been fixed. Could someone merge this? @tracykteal @raynamharris ?

**Review by mkuzak — APPROVED** [review]

**Review by ChristinaLK — APPROVED** [review]

**Comment by jduckles** [comment]
All checks/reviews have passed *looks up at that shiny green button* who wants to be "the one" to merge? ;) @raynamharris

File: RFCs/RFC1.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+Carpentry (Lesson Organization) is primarily responsible for lesson development and maintenance. 
+(Approved [July 3](https://github.com/swcarpentry/board/blob/master/minutes/minutes-2017-07-03.md))*
+
+## Current vision of responsibilities for The Carpentries:
+* Operations:
+  * Coordinating with partner organizations and grant agencies (negotiating contracts, allocation of resources, 
+  	fulfillment of obligations)
+  * Maintaining budget and reporting on finances
+  * Appropriately allocating staff time to meet the needs of The Carpentries community and lesson organizations
+  * Managing duplication and overlap (minimizing competition and mediating conflict between lesson organizations)
```
Comment (tracykteal) [inline RFCs/RFC1.md:19]: * Instructor membership management and maintaining records of Carpentries instructors

File: RFCs/RFC2.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+* *The Board of Directors for the combined organization [The Carpentries] will be composed of 9 members, each serving 
+a two year term without limits on the number of terms. Five members of the board will be appointed through a process 
+of nomination (with community input) to the board followed by voting by remaining board members. The other four members 
+will be elected by the membership of the organization. 
+(Approved [July 3](https://github.com/swcarpentry/board/blob/master/minutes/minutes-2017-07-03.md))*
+* *Appointed members: The first set of appointees come from current members of DC’s Board (Ethan White and Karen Cranston) 
+and SWC’s (elected) Steering Committee (Mateusz Kuzak, Sue McClatchy, Kate Hertweck). These individuals were chosen to 
+represent our current leadership perspectives from multiple viewpoints (geography, technical expertise, experience with 
+the Carpentries). These first appointments have been approved by the motion of DC Board and SWC Steering Committee on
```
Comment (tracykteal) [inline RFCs/RFC2.md]: DC Board -> DC Steering Committee

File: RFCs/RFC5.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
+# RFC5: Finances
+
+## Current vision for The Carpentries finances:
+* Software Carpentry (SWC) and Data Carpentry (DC) are currently separate financial entities with independent budgets, 
+both sheltered by our fiscal sponsor NumFOCUS. 
+* Most income for both organizations is derived from organizational membership agreements (e.g., contracts signed 
+with institutions as silver/platinum memberships), grants, and workshop fees. The vast majority of organizational
```
Comment (tracykteal) [inline RFCs/RFC5.md]: silver/platinum -> silver/gold/platinum

File: RFCs/RFC5.md [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+
+## Current vision for The Carpentries finances:
+* Software Carpentry (SWC) and Data Carpentry (DC) are currently separate financial entities with independent budgets, 
+both sheltered by our fiscal sponsor NumFOCUS. 
+* Most income for both organizations is derived from organizational membership agreements (e.g., contracts signed 
+with institutions as silver/platinum memberships), grants, and workshop fees. The vast majority of organizational 
+memberships are joint between SWC and DC, so institutional members are not likely to experience substantially different 
+interactions with The Carpentries than with SWC and DC as they currently exist. 
+* The Carpentries would operate under a unified budget initially sheltered under NumFOCUS, but will initiate planning 
+for emergence as an independent 501(c)(3) non-profit organization registered in the USA.
```
Comment (tracykteal) [inline RFCs/RFC5.md]: We may not necessarily be planning to be our own 501(c)3 in the short term. I think we could leave this at 'initially sheltered under NumFOCUS.'

File: RFCs/RFC6.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+be adapted to suit The Carpentries. Once approved by The Carpentries Board of Directors, each subcommittee is responsible 
+for determining their own method of governance (e.g., who is a member, meeting schedule) to best meet their needs, although 
+a point of contact (usually chair of the subcommittee) is required for reporting purposes.
+* Existing (active) subcommittees and task forces (all planned to continue): 
+  * Instructor trainers: conducting instructor training, maintaining lesson repo (movement to Carpentries organization 
+  approved by SWC Steering Committee)
+  * Policy: enforcing Code of Conduct
+  * CarpentryCon task force: planning and implementing (yearly?) convention for community
+  * Mentoring: resource for instructors and connected with instructor training
+  * Infrastructure: maintaining consistent and stable templating for lessons
```
Comment (tracykteal) [inline RFCs/RFC6.md]: for lessons -> for lessons and workshops?

File: RFCs/RFC7.md [hunk 11]
```diff
@@ -0,0 +1,24 @@
+# RFC7: Lesson Organizations
+
+## Current vision for Lesson Organizations:
+* Responsibilities:
```
Comment (tracykteal) [inline RFCs/RFC7.md]: Responsibilities -> Responsibilities of the Lesson Organizations:

File: RFCs/RFC7.md [hunk 12]
```diff
@@ -0,0 +1,24 @@
...
+  maintaining lesson repositories in GitHub based on the lesson example from The Carpentries GitHub organization, 
+  determining processes associated with lesson maintenance (recommendations for lesson incubation and maintenance?)
+* Governance: Each lesson organization should have a point of contact (preferably a committee of 5-7 people led by a Chair), 
+although the specific methods through which leadership occurs is determined by the lesson organization.
+Onboarding for new lesson organizations: The Carpentries Board will be responsible for assessing alignment of the prospective 
+lesson organization’s mission and their ability to meet the responsibilities described. Given these decisions impact 
+organizational resources, the Board of Directors will solicit community input prior to deciding whether to extend an offer. 
+The Carpentries Board may also recommended compatible administrative structures and policies for prospective Lesson 
+Organizations.
+
```
Comment (tracykteal) [inline RFCs/RFC7.md]: **Responsibilities of the Carpentries to the Lesson Organizations:**
These will continue to be developed, but include the general categories of:
* Workshop coordination, including infrastructure
* Lesson and workshop templates
* Lesson development infrastructure
* Assessment
* Instructor training
* Community support

File: RFCs/RFC7.md [hunk 13]
```diff
@@ -0,0 +1,24 @@
...
+## For discussion:
+
+Please add your comments on RFC7 to [this issue](#7).
+* Q1: Do you have any comments or suggestions on the organisation and role of the Lesson Organisations under the umbrella of The Carpentries organisation? Specifically:
+	* Q1a: responsibilities to The Carpentries and to the community?
+	* Q1b: governance model?
```
Comment (tracykteal) [inline RFCs/RFC7.md:33]: Q1c: responsibilities of the The Carpentries to the Lesson Organizations

**Comment by tracykteal** [comment]
Thanks for all of this, and I'm sorry for my delayed comments. I made a few comments. The most significant is in RFC7 where I added a section on responsibilities of the Carpentries to the Lesson Organizations. I think it's important to start out by showing that there are obligations in both directions. There are still ideas to be worked out there, so I added general categories rather than concrete suggestions. Does this seem OK?

**Comment by tracykteal** [comment]
On all the RFCs, we might consider adding 

STATUS: Open or STATUS: Open until October 6, 2017 to the top

Then we can also ultimately have a RFC landing page with "Open" and "Prior" categories, like in https://www.hhs.gov/ohrp/regulations-and-policy/requests-for-comments/index.html

**Comment by k8hertweck** [comment]
Thanks for the comments, @tracykteal, I think I've incorporated all of them.

Could someone please merge this PR? @raynamharris? It may also be a good idea for me to have push/merge permission as well, so I can update if necessary.

**Pull request merged**